pkg/cmd/kafka/update/update.go

```diff
@@ -119,7 +118,6 @@ func NewUpdateCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", dump.JSONFormat, opts.localizer.MustLocalize("kafka.common.flag.output.description"))
```

Comment:
Similar. What is the reason for leaving outputFormat in options?

Reply:
I just forgot tbh. Will remove.